controllers/api/picture.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+//const {  } = require("../models");
+
```

Comment:
This file has the wrong path. Should be: `controllers/api/picture/index.js`
Here you would import and route:
- [ ] `./comment.js` @ `/:imageId/comment`
- [ ] `./like.js` @ `/:imageId/like`
- [ ] `./gallery.js` @ `/:imageId/gallery`